auth/src/main/java/com/firebase/ui/auth/ui/FragmentHelper.java

```diff
@@ -6,6 +6,7 @@
 import android.os.Bundle;
 import android.support.annotation.RestrictTo;
 import android.support.v4.app.Fragment;
+import android.support.v7.view.ContextThemeWrapper;
```

Comment:
Oops, I'm so used to defaulting to support lib classes that I don't even think twice about it. 😄